Grenache.Utils/Grenache.Utils.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
hmm afaik we removed newtonsoft in favor of default dotnet libs, can we use those please? see https://github.com/bitfinexcom/grenache-csharp/pull/1/commits/616015265866f2040e61b74e9995948ebd39769a

Reply:
done